appcues/src/main/java/com/appcues/debugger/DebuggerRecentEventsManager.kt

```diff
@@ -111,12 +114,21 @@ internal class DebuggerRecentEventsManager(
         lastEventId++
     }
 
+    private fun ActivityRequest.getIdentifyTimeStamp(): Long {
+        return if (!appcuesConfig.isTesting) {
+            // Unsure why we need to do this, we could just get current date or ActivityRequest.timeStamp
```

Comment:
I don't know either, it seems like the ActivityRequest timestamp would be accurate for this.

Reply:
not sure why we do it like this, I think we can clean it up and use the activity request timestamp instead too.